components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/primary/PrimaryCardView.kt

```diff
@@ -15,4 +15,64 @@
  */
 package uk.gov.hmrc.components.compose.organism.primary
 
-class PrimaryCardView
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import uk.gov.hmrc.components.compose.atom.heading.Heading5
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+
+@Composable
+fun PrimaryCard(
+    modifier: Modifier = Modifier,
+    title: String,
+    titleContentDescription: String = "",
+    childPadding: Boolean = true,
+    content: @Composable (ColumnScope.() -> Unit)? = null
+) {
+    val childPaddingState by remember { derivedStateOf { childPadding } }
+
+    Column(
+        modifier = modifier
+            .background(HmrcTheme.colors.hmrcWhiteBackground)
+            .fillMaxWidth()
+    ) {
+        Heading5(
+            text = title,
+            modifier = Modifier
+                .padding(horizontal = HmrcTheme.dimensions.hmrcSpacing16)
+                .padding(top = HmrcTheme.dimensions.hmrcSpacing16)
```

Comment:
Feels a bit odd to have two different padding modifiers declared.. can we combine them?